Makefile

```diff
@@ -92,9 +92,12 @@ gencatalog:
 	(cd site/content/en/guides/consumer/function/catalog/catalog && npm run gen-docs)
 
 site-run-server:
-	npx http-server site/ -p 3000
+	setfacl -Rd -m o::rx site/ && chmod o+rx -R site/
```

Comment:
this is a linux-specific command. Our developer infra should work on at least mac.

Reply:
Added `chmod -R +a ...` for mac.

Reply:
```
 ~/src/kpt » setfacl
zsh: command not found: setfacl
```
?

Reply:
Yes, the `chmod` flags for ACLs don't exist on Linux. @mikebz That would be expected on Mac but the ACL would still be set due to the second half of the line. Since it's expected that one of the two commands will complete, suppressed the error of `setfacl` not existing so there's no error output on Mac.

Reply:
My question is what is the advantage of that command given that you are doing `o::rx` with it, basically doing a chmod anyway...

Reply:
The advantage is to allow files created while the docker image is up to be picked up by the running site instance